web/packages/api/src/assets_v2.ts

```diff
@@ -1154,6 +1203,178 @@ function addOverrides(envName: string, result: RegistryOptions) {
+                        symbol: "KSM",
+                        decimals: 12,
+                        isSufficient: true,
+                        location: {
+                            parents: 2,
+                            interior: {
+                                x1: [
+                                    {
+                                        globalConsensus: "Kusama",
+                                    },
+                                ],
+                            },
+                        },
+                    },
+                    {
+                        token: "0x21FaB0eA070F162180447881D5873Cf3d57200d6".toLowerCase(),
+                        name: "Kolkadot",
+                        minimumBalance: 1n,
+                        symbol: "KOL",
+                        decimals: 12,
+                        isSufficient: false,
+                        location: {
+                            parents: 0,
+                            interior: { X2: [{ palletInstance: 50 }, { generalIndex: 86 }] },
+                        },
+                        assetId: "86",
+                    },
+                    {
+                        token: "0x92262680A8d6636bbA9bFFDf484c274cA2de6400".toLowerCase(),
+                        name: "DED",
+                        minimumBalance: 1n,
+                        symbol: "DED",
+                        decimals: 10,
+                        isSufficient: false,
+                        location: {
+                            parents: 0,
+                            interior: { X2: [{ palletInstance: 50 }, { generalIndex: 30 }] },
+                        },
+                        assetId: "30",
+                    },
+                    {
+                        token: "0xa37B046782518A80e2E69056009FBD0431d36E50".toLowerCase(),
+                        name: "PINK",
+                        minimumBalance: 1n,
+                        symbol: "PINK",
+                        decimals: 10,
+                        isSufficient: false,
+                        location: {
+                            parents: 0,
+                            interior: { X2: [{ palletInstance: 50 }, { generalIndex: 23 }] },
+                        },
+                        assetId: "23",
+                    },
+                    {
+                        token: "0x5FDcD48F09FB67de3D202cd854B372AEC1100ED5".toLowerCase(),
+                        name: "GAVUN WUD",
+                        minimumBalance: 1n,
+                        symbol: "WUD",
+                        decimals: 10,
+                        isSufficient: false,
+                        location: {
+                            parents: 0,
+                            interior: { X2: [{ palletInstance: 50 }, { generalIndex: 31337 }] },
+                        },
+                        assetId: "31337",
+                    },
+                ],
+            }
```

Comment:
Why does asset discovery not pick up these assets?

Reply:
The reason mentioned in https://linear.app/snowfork/issue/SNO-1207#comment-de6709cc